tools/launcher/common/specdec_bench/run.sh

```
@@ -0,0 +1,53 @@
+# silently drifting versions if upstream wheels move between launches.
+if ! pip show boto3 >/dev/null 2>&1 || \
+   ! pip show datasets >/dev/null 2>&1 || \
+   ! pip show seaborn >/dev/null 2>&1; then
+    if ! pip install -r modules/Model-Optimizer/examples/specdec_bench/requirements_speed.txt; then
+        report_result "FAIL: specdec_bench: pip install requirements_speed.txt failed"
+        exit 1
+    fi
+fi
+
+if ! python3 modules/Model-Optimizer/examples/specdec_bench/run.py \
+    --model_dir ${HF_MODEL_CKPT} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)